awsmfa/config.py

```diff
@@ -25,8 +26,5 @@ def initial_setup(logger, config, config_path):
     if aws_secret_access_key is None or aws_secret_access_key == "":
         log_error_and_exit(logger, "You must supply aws_secret_access_key")
 
-    config.add_section(profile_name)
-    config.set(profile_name, 'aws_access_key_id', aws_access_key_id)
-    config.set(profile_name, 'aws_secret_access_key', aws_secret_access_key)
-    with open(config_path, 'w') as configfile:
-        config.write(configfile)
+    keyring.set_password("aws:access_key_id", profile_name, aws_access_key_id)
+    keyring.set_password("aws:secret_access_key", profile_name, awsgetpass.getpass_secret_access_key)
```

Comment:
just `aws_secret_access_key` here, you've already gotten the value with `getpass` earlier

Reply:
This code is responsible for putting credentials into keychain, not for getting them.

Reply:
Yeah but check the value you are trying to set. it should be `aws_secret_access_key` as mentioned above (and not `awsgetpass.getpass_secret_access_key`)

Reply:
Got it! accidental string paste.